linera-base/src/crypto/mod.rs

```diff
@@ -35,6 +36,11 @@ pub type ValidatorSignature = secp256k1::Secp256k1Signature;
 /// The key pair of a validator.
 pub type ValidatorKeypair = secp256k1::Secp256k1KeyPair;
 
+/// The SECP256k1 public key sizes also used for validators.
+pub const SECP256K1_PUBLIC_KEY_SIZE: usize = std::mem::size_of::<secp256k1::Secp256k1PublicKey>();
+/// The SECP256k1 signature sizes also used for validators.
+pub const SECP256K1_SIGNATURE_SIZE: usize = std::mem::size_of::<secp256k1::Secp256k1Signature>();
+
```

Comment:
Actually, do we still need those at all? Aren't those "simple", too?

(And even if: If it's only used in one place I'd rather use `size_of` there directly?)